### PR TITLE
translate docs/api-reference/v1.5/README.md into chinese

### PR DESCRIPTION
**Pull request by lmxia**

### CODE REVIEW

File: docs/api-reference/v1.5/README.md [hunk 1]
```diff
@@ -26,44 +40,93 @@ spec:
...
+
 ```
 npm start
 ```
 
+<!--
 ## License
+-->
+
+##许可证
```
Comment (chentao1596) [inline docs/api-reference/v1.5/README.md]: 同上：##后面需要有空格，否则无法正常展示格式

File: docs/api-reference/v1.5/README.md [hunk 2]
```diff
@@ -26,44 +40,93 @@ spec:
...
 
+bdocs-tab:tab_lang 用于指示这些代码段属于哪个选项卡。tab 部分的字符串指示了选项卡,而语言被放到了下划线之外。在渲染过程中，语言将正确高亮，就好像bdoc标记被省略一样。
+
+<!--
 ## Motivation
 
 This is a project to extend markdown documents and render them in html with a table of contents and code snippet pane. Most projects of this variety lean heavily on front end parsing with JavaScript/jQuery. This project uses NodeJS, Marked, and highlight.js to output syntax highlighted code blocks.
+-->
+
+##动机
```
Comment (chentao1596) [inline docs/api-reference/v1.5/README.md]: ##后面需要有空格，否则无法正常展示格式
```
## 动机
```

**Review by chentao1596 — CHANGES_REQUESTED** [review]
Review finished, ^_^!

**Comment by lmxia** [comment]
@chentao1596 modified please check.

**Review by chentao1596 — APPROVED** [review]
LGTM
@markthink @zhangxiaoyu-zidif

**Comment by zhangxiaoyu-zidif** [comment]
the markdown format may be wrong.
Many lines are not been bootstrap.

**Comment by lmxia** [comment]
@zhangxiaoyu-zidif  You are right, thanks for your review, please check agin.

**Pull request merged**